pkg/util/managedcluster/managedcluster.go

```diff
@@ -42,3 +46,47 @@ func ClientsetFromConfig(cs *api.OpenShiftManagedCluster) (*kubernetes.Clientset
 
 	return kubernetes.NewForConfig(restconfig)
 }
+
+// getKubeconfigFromV1Config takes a v1 config and returns a kubeconfig
+func getKubeconfigFromV1Config(kc *v1.Config) (clientcmd.ClientConfig, error) {
```

Comment:
this is now practically identical to ClientsetFromConfig above and can presumably go away?

Reply:
i'll either fix this or raise an issue to track it.